### PR TITLE
Save hd5

### PR DESCRIPTION
**Pull request by eisascience**
a new fx mostly for the older ser objs saved as .rds files. the new .rds files are 1:1 with the hd5 version.

### CODE REVIEW

File: R/SeuratUtils.R [hunk 1]
```diff
@@ -358,3 +358,37 @@ FeaturePlotAcrossReductions <- function(seuratObj, features, reductions = c('tsn
...
+    so = readRDS(load.path)
+    print("load of RDS from drives complete")
+  }
+  
+  if(updateSerObj) so = SeuratDisk::UpdateSeuratObject(so)
+  
+  if(is.null(save.path)) save.path = gsub(".rds", "hd5Ser", load.path)
+  print("saving file path:")
+  print(save.path)
+  SeuratDisk::SaveH5Seurat(so, filename=save.path,  overwrite = overwrite)
```
Comment (bbimber) [inline R/SeuratUtils.R:390]: wouldnt the package need to declare a dependency on SeuratDisk?

File: R/CiteSeq.R [hunk 2]
```diff
@@ -712,4 +712,36 @@ RunSeuratWnn <- function(seuratObj, dims.list = list(1:30, 1:18), reduction.list
...
+  
+  KeepCells <- colnames(DPE.mat)[colSums(DPE.mat)>min.thresh]
+  
+  if(!is.null(max.thresh)){
+    DPE.mat = DPE.mat[,KeepCells]
+    KeepCells <- colnames(DPE.mat)[colSums(DPE.mat)<max.thresh]
+  }
+  
+  print("Subsetting the cells by ADT")
+  return(subset(seuratObj, cell.barcode %in% KeepCells))
```
Comment (bbimber) [inline R/CiteSeq.R:746]: didnt we talk about this? how is this better than subset(nCount_ADT > X)?

**Review by bbimber — CHANGES_REQUESTED** [review]

File: R/CiteSeq.R (hunk 2 above)
Comment (eisascience) [reply R/CiteSeq.R:746]: sorry this was a lingering update I didnt clear

File: R/SeuratUtils.R (hunk 1 above)
Comment (eisascience) [reply R/SeuratUtils.R:390]: yes we need to change the NAMESPACE file too

**Pull request closed without merging**